performant-software/DM
# Text manipulation fixes

### What does this PR do?
Fixes issues where duplicated/lingering elements were causing failures to show or hide text.

### What issues does it address?
Addresses #108, #110, #122 

### How to test
Fixes have been pushed to [staging](http://67.207.158.85/workspace/#6kgzvltt). To test #108, add a text resource to the grid and quickly (before "Loading..." is replaced by the actual text) drag and drop it to a new spot in the grid. For #110 / #122, open a resource in one project and pull up the rollover window for an annotation that shows at least one image resource target. Then switch projects, switch back to the first project, and do the same. Check to see if the image titles display correctly in the rollover window.
